ClavaLaraApi/src-lara-clava/clava/clava/ClavaJoinPoints.lara

```diff
@@ -483,3 +483,24 @@ ClavaJoinPoints.forStmt = function($init, $condition, $inc, $body) {
 ClavaJoinPoints.param = function(name, $type) {
 	return AstFactory.param(name, $type);
 }
+
+
+/**
+ * @param {string} text
+ *
+ * @return {$comment}
+ * */
+ClavaJoinPoints.comment = function(text) {
+    
+}
+
+/**
+ * @param {$joinpoint} $jp
+ *
+ * @return {$joinpoint}
+ * */
+ClavaJoinPoints.makeComment = function($jp) {
+    return $jp.replaceWith(
+        ClavaJoinPoints.comment($jp.code)
+    );
```

Comment:
```suggestion
```

Reply:
I've added a global action $jp.toComment, with overloads .toComment(prefix) and toComment(prefix, suffix).